### PR TITLE
Alert avatar

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

File: alert/alert-dark.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    margin: 1rem 0;
+}
+
+.alert::before{
+    content:"";
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    border: solid 2px currentColor;
```
Comment (rahulyadav139) [inline alert/alert-dark.css:40]: You miss to use var(--currentColor). I am assuming currentColor is a CSS variable.

File: alert/alert-dark.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+.alert-themed{
+    color: var(--color-primary);
+    background: var(--color-primary);
+    background: var(--primary-themed-gradient);
+}
+
+.alert-outlined{
+    background: transparent;
+    border: 1px solid currentColor;
+}
```
Comment (rahulyadav139) [inline alert/alert-dark.css:82]: Use var keyword to call a CSS variable.

File: avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+.avatar-m{
+    width: 6rem;
+    height: 6rem;
+    font-size: var(--fsize-400);
+}
+
+.avatar-l{
+    width: 8rem;
+    height: 8rem;
```
Comment (rahulyadav139) [inline avatar/avatar.css:32]: Great naming convention.

File: darktheme.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    --box-shadow: 0 14px 8px -2px rgba(255, 255, 255, 0.3);
+
+    /* font-size */
+    --fsize-700: 4rem;
+    --fsize-600: 3rem;
+    --fsize-500: 2rem;
+    --fsize-400: 1.5rem;
+    --fsize-300: 1.25rem;
+    --fsize-200: 1rem;
+    --fsize-100: 0.875rem;
```
Comment (rahulyadav139) [inline darktheme.css:40]: This is very convenient. You defined all useable font size one place. Awesome approach.

**Review by aiman-ansari — APPROVED** [review]
Code is clean and neat .
👍

File: alert/alert-dark.css (hunk 1 above)
Comment (muskanbagrecha) [reply alert/alert-dark.css:40]: currentColor is a CSS keyword referring to the current value of the color property

File: avatar/avatar.css (hunk 3 above)
Comment (muskanbagrecha) [reply avatar/avatar.css:32]: Thank you

File: darktheme.css (hunk 4 above)
Comment (muskanbagrecha) [reply darktheme.css:40]: Thank you

**Comment by ShrutiLakhani** [comment]
Good work! Appreciate how you have pre defined the font sizes and other reusable properties.

**Pull request merged**